tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -1,258 +1,212 @@
-    expect(button).toHaveLength(1);
-    expect(button).toHaveProp('logInText', 'Log in to view your collections');
+    expect(
+      screen.getByRole('link', { name: 'Log in to view your collections' }),
+    ).toBeInTheDocument();
   });
 
   it('does not render an AuthenticateButton with a logged in user', () => {
-    const { store } = dispatchSignInActions();
-    const root = renderComponent({ store });
+    signInUser();
+    render();
 
-    expect(root.find(AuthenticateButton)).toHaveLength(0);
+    expect(
+      screen.queryByRole('link', { name: 'Log in to view your collections' }),
+    ).not.toBeInTheDocument();
   });
 
   it('renders the collection listing info card', () => {
-    const { store } = dispatchSignInActions();
-    const root = renderComponent({ store });
-
-    expect(root.find('.CollectionList-info')).toHaveLength(1);
-    expect(root.find('.CollectionList-info')).toHaveProp(
-      'header',
-      'Collections',
-    );
-    expect(root.find('.CollectionList-info-text'))
-      .toHaveText(oneLine`Collections make it easy to keep track of favorite
-      add-ons and share your perfectly customized browser with others.`);
-
-    const button = root.find('.CollectionList-create');
-    expect(button).toHaveProp('buttonType', 'action');
-    expect(button).toHaveProp('puffy', true);
-    expect(button).toHaveProp('to', '/collections/add/');
-    expect(button.childAt(0)).toHaveText('Create a collection');
+    signInUser();
+    render();
+
+    expect(screen.getByText('Collections')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        `Collections make it easy to keep track of favorite add-ons and ` +
+          `share your perfectly customized browser with others.`,
+      ),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('link', { name: 'Create a collection' }),
+    ).toHaveAttribute('href', `/${lang}/${clientApp}/collections/add/`);
   });
 
   it('renders placeholder text if the user has no collections', () => {
-    const userId = 1234;
-    const { store } = dispatchSignInActions({ userId });
-
-    store.dispatch(
-      loadUserCollections({
-        collections: [],
-        userId,
-      }),
-    );
-
-    const root = renderComponent({ store });
-
-    expect(root.find('.CollectionList-list')).toHaveLength(1);
-    expect(root.find('.CollectionList-list')).toHaveProp(
-      'header',
-      'My collections',
-    );
-    expect(root.find('.CollectionList-list')).toHaveProp(
-      'footer',
-      'You do not have any collections.',
-    );
-    expect(root.find('.CollectionList-listing')).toHaveLength(0);
+    signInUser();
+    store.dispatch(loadUserCollections({ collections: [], userId }));
+    render();
+
+    expect(screen.getByText('My collections')).toBeInTheDocument();
+    expect(
+      screen.getByText('You do not have any collections.'),
+    ).toBeInTheDocument();
+    // TODO: Assert that no items are displayed.
```

Comment:
is this a TODO for a follow-up or you forgot to finish?

Reply:
It was a reminder to myself to come back to this. Thanks for catching that. Let me see if it is actually needed.